### PR TITLE
Test pet model

### PR DESCRIPTION
**Pull request by shofaizi**
@whilestevego

### CODE REVIEW

**Review by jeffsmw2 — APPROVED** [review]
👍

File: Gemfile [hunk 1]
```diff
@@ -60,9 +60,10 @@ gem 'font-awesome-rails'
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platform: :mri
-
+  gem "rspec-rails", :group => [:development, :test]
```
Comment (whilestevego) [inline Gemfile]: This `gem` is already in the `:development` and `:test` group. There is no need to specify it twice. Also, other gems use `'`. Don't use `"`

File: spec/models/message_spec.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (whilestevego) [inline spec/models/message_spec.rb]: Remove this empty line

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  #     end
+  #
+  # The different available types are documented in the features, such as in
+  # https://relishapp.com/rspec/rspec-rails/docs
+  config.infer_spec_type_from_file_location!
+
+  # Filter lines from Rails gems in backtraces.
+  config.filter_rails_from_backtrace!
+  # arbitrary gems may also be filtered via:
+  # config.filter_gems_from_backtrace("gem name")
```
Comment (whilestevego) [inline spec/rails_helper.rb]: Get rid of all the comments in this file! 🔥🔥🔥🔥🔥

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  # the seed, which is printed after each run.
+  #     --seed 1234
+  config.order = :random
+
+  # Seed global randomization in this process using the `--seed` CLI option.
+  # Setting this allows you to use `--seed` to deterministically reproduce
+  # test failures related to randomization by passing the same `--seed` value
+  # as the one that triggered the failure.
+  Kernel.srand config.seed
+=end
```
Comment (whilestevego) [inline spec/spec_helper.rb]: 🔥🔥🔥🔥 the comments!

**Review by whilestevego — CHANGES_REQUESTED** [review]

**Comment by shofaizi** [comment]
@whilestevego  took care of the changes pls approve

**Comment by whilestevego** [comment]
@shofaizi why did you close the PR?

**Comment by shofaizi** [comment]
@whilestevego

**Comment by whilestevego** [comment]
@shofaizi solve the conflicts. Come see me if you're unsure how 😎

**Comment by whilestevego** [comment]
This branch is irrelevant. Pet tests were added as part of https://github.com/CodeCoreYVR/find_my_pet/pull/7

**Pull request closed without merging**